plugins/cost-insights/src/hooks/useFilters.tsx

```diff
@@ -69,6 +69,7 @@ export const FilterContext = React.createContext<
 >(undefined);
 
 export const FilterProvider = ({ children }: FilterProviderProps) => {
+  const config = useConfig();
```

Comment:
Looks like we already have ` const { products } = useConfig();` lower in the file

Reply:
Didn't apply the style change globally but can update